src/libraries/Microsoft.PowerFx.Core/Public/Config/ComposedReadOnlySymbolTable.cs

```diff
@@ -136,6 +136,11 @@ public virtual bool Lookup(DName name, out NameLookupInfo nameInfo, NameLookupPr
             return false;
         }
 
+        internal override bool TryLookup(DName name, out NameLookupInfo nameInfo)
```

Comment:
>TryLookup [](http://example.com/codeflow?start=31&length=9)

What's the relationship here to Lookup() on 125?

Reply:
Lookup is doing a TryLookup but isn't wired to it in the base class which always return false

Reply:
I'll remove it